src/java.base/share/classes/java/security/Provider.java

```diff
@@ -1311,7 +1311,7 @@ public Service getService(String type, String algorithm) {
     // re-use will occur e.g. as the framework traverses the provider
     // list and queries each provider with the same values until it finds
     // a matching service
-    private static volatile ServiceKey previousKey =
+    private transient ServiceKey previousKey =
                                             new ServiceKey("", "", false);
```

Comment:
Please fix the formatting while you are making the change.  If a second line is still needed, it should be 4 whitespaces from the left margin.  The formatting was wrong before your change.  Thanks